### PR TITLE
update: release notes and tutorials

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

**Review by yalishevant — APPROVED** [review]

File: docs/topics/jvm/jvm-get-started.md [hunk 1]
```diff
@@ -4,23 +4,33 @@ This tutorial demonstrates how to use IntelliJ IDEA for creating a console appli
...
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location if necessary.
+4. Select the **Create Git repository** checkbox to place the new project under version control. You will be able to do it
+later at any time.
+5. From the **Language** list, select **Kotlin**.
    
-   By default, your project will use the Gradle build system with Kotlin DSL.
+   ![Create a console application](jvm-new-project.png){width=700}
+
+6. Select the **IntelliJ** build system. It's a native builder that doesn't require downloading additional artifacts.
```
Comment (koshachy) [inline docs/topics/jvm/jvm-get-started.md:18]: Why IntelliJ?
Usually we don't promote it

Comment (koshachy) [reply docs/topics/jvm/jvm-get-started.md:18]: I don't think it's useful when we explain how to start a project with IntelliJ build system and then everywhere talk only about the Gradle/Maven.
What do you think?

File: docs/topics/jvm/jvm-get-started.md [hunk 2]
```diff
@@ -4,23 +4,33 @@ This tutorial demonstrates how to use IntelliJ IDEA for creating a console appli
...
+   If you want to create a more complex project that needs further configuration, select Maven or Gradle. For Gradle,
+   choose a language for the build script: Kotlin or Groovy.
+7. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+   * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+   JDK home directory. 
+   * If you don't have the necessary JDK on your computer, select **Download JDK**.
 
-4. Go through and accept the default configuration, then click **Finish**. Your project will open.
+8. Enable the **Add sample code** option to create a file with a sample `Hello World!` application.
```
Comment (koshachy) [inline docs/topics/jvm/jvm-get-started.md:28]: Why **Hello World!** in the backticks?

**Review by koshachy — CHANGES_REQUESTED** [review]
Double-check the **releases.md** file.
It has also information and links to the newest IDEA and plugin.

**Review by koshachy — APPROVED** [review]
Discussed.
Approved, still requires small fixes.

**Pull request merged**